packages/fullstack/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cspell-dict-fullstack",
-  "version": "1.0.13",
+  "version": "1.0.14",
```

Comment:
No need to change the version, it gets automatically updated during the publish phase.

Reply:
Ok, should I downgrade the version?

Reply:
Please leave it at `1.0.13`. When I publish the dictionary it will automatically get updated.